looker_deployer/commands/deploy_content.py

```diff
@@ -125,8 +133,56 @@ def import_content(content_type, content_json, space_id, env, ini, debug=False):
         win_exec = ["cmd.exe", "/c"]
         gzr_command = win_exec + gzr_command
 
+    is_new_dash = "false"
+    existing_dash_alerts = []
+    existing_elements_w_alerts = []
+
+    if content_type == "dashboard":  ## only run for dashboards, looks can't have alerts
+        sdk = get_client(ini, env)  ## should we update the function def and pass sdk?
+        with open(content_json) as file:
+            json_dash = json.load(file)
+        existing_dash = sdk.search_dashboards(slug=json_dash['slug'])  ## search with slug first, fall back to name + folder
+        if len(existing_dash) < 1:
+            existing_dash = sdk.search_dashboards(title=json_dash['title'],folder_id=space_id)
+            if len(existing_dash) < 1:
+                is_new_dash = "true"
+        if is_new_dash == "false": ## if it's an existing dashboard, save the alerts and elements
+            for element in existing_dash[0]['dashboard_elements']:
+                start = len(existing_dash_alerts)
+                alerts = list(filter(lambda alert: alert['dashboard_element_id'] == element['id'], enabled_alerts))
```

Comment:
To get this to work I needed to do the following `str(alert['dashboard_element_id']) == str(element['id'])` I think the alert ID might be a number 🤦 this allowed for the alert to be found